contracts/FlowIDTableStaking.cdc

```diff
@@ -531,6 +555,8 @@ pub contract FlowIDTableStaking {
 
                 // update request to show that leftover amount is requested to be unstaked
                 nodeRecord.tokensRequestedToUnstake = nodeRecord.tokensRequestedToUnstake + (amount - amountCommitted)
+
```

Comment:
I would like to see an Event here for a requested unstake so that alerting / tracking can be done proactively for operators making these moves.

Reply:
added in [779898a](https://github.com/onflow/flow-core-contracts/pull/324/commits/779898a7332503673de661051664e30e7fc0950a)